### PR TITLE
Cincinnati: use `buildroot` instead of `src` as base for checks

### PR DESCRIPTION
**Pull request by vrutkovs**
Buildroot caching makes updates complicated, so this PR would switch those tests to use a newly built `buildroot` image as base

### CODE REVIEW

**Comment by vrutkovs** [comment]
/cc @LalatenduMohanty 

/hold
until rehearsals pass

**Comment by vrutkovs** [comment]
This seems to get us closer to functioning builds - this would unblock https://github.com/openshift/cincinnati/pull/361

/hold cancel

**Review by LalatenduMohanty — APPROVED** [review]
/lgtm

**Pull request merged**
